api/client/gro_client.py

```diff
@@ -217,6 +217,7 @@ def find_data_series(self, **kwargs):
         See also
         --------
         get_data_series()
+        https://developers.gro-intelligence.com/data-series-definition.html
```

Comment:
I'd removed this here https://github.com/gro-intelligence/api-client/commit/3099c393d67e8c2870d1086606673112756b5abd because the formatting for the link doesn't work in the "See also" section. See the line-break here:
![Screen Shot 2019-12-09 at 8 58 26 PM](https://user-images.githubusercontent.com/3478393/70472227-ac9c6980-1ac6-11ea-8adc-1a07ae0899f6.png)

Reply:
^ This was still there, so I just pushed https://github.com/gro-intelligence/api-client/pull/140/commits/82b83d8a600578a0842088ab1875ff8784efe698 to remove